### PR TITLE
chore(cli): display server's time as default

### PR DESCRIPTION
**Pull request by sundy-li**

### CODE REVIEW

**Review by TCeason — APPROVED** [review]

File: .github/actions/check/action.yml [hunk 1]
```diff
@@ -32,10 +32,10 @@ runs:
...
         cargo machete
 
-    - name: Deny Check
-      shell: bash
-      run: |
-        cargo deny check
+    # - name: Deny Check
+    #   shell: bash
+    #   run: |
+    #     cargo deny check
```
Comment (everpcpc) [inline .github/actions/check/action.yml:38]: We should figure out the error instead of just disabling the check 😢

Comment (sundy-li) [reply .github/actions/check/action.yml]: do you have any ideas to fix:

```
error[unmaintained]: proc-macro-error is unmaintained
    ┌─ /home/runner/work/bendsql/bendsql/Cargo.lock:272:1
    │
272 │ proc-macro-error 1.0.4 registry+https://github.com/rust-lang/crates.io-index
    │ ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ unmaintained advisory detected
    │
    ├ ID: RUSTSEC-2024-0370
    ├ Advisory: https://rustsec.org/advisories/RUSTSEC-2024-0370
    ├ proc-macro-error's maintainer seems to be unreachable, with no commits for 2 years, no releases pushed for 4 years, and no activity on the GitLab repo or response to email.
      
      proc-macro-error also depends on `syn 1.x`, which may be bringing duplicate dependencies into dependant build trees.
      
      ## Possible Alternative(s)
      
      - [manyhow](https://crates.io/crates/manyhow)
      - [proc-macro-error2](https://crates.io/crates/proc-macro-error2)
      - [proc-macro2-diagnostics](https://github.com/SergioBenitez/proc-macro2-diagnostics)
    ├ Announcement: https://gitlab.com/CreepySkeleton/proc-macro-error/-/issues/20
    ├ Solution: No safe upgrade is available!
    ├ proc-macro-error v1.0.4
      ├── fastrace-macro v0.6.8
      │   └── fastrace v0.6.8
      │       └── databend-common-ast v0.0.3
      │           └── bendsql v0.20.1
      └── nom-rule v0.3.1
          └── databend-common-ast v0.0.3 (*)
```

Comment (everpcpc) [reply .github/actions/check/action.yml]: It seems the dependency for `databend-common-ast` should be upgraded

Comment (sundy-li) [reply .github/actions/check/action.yml]: There is no way to upgrade crate `proc-macro-error`, it's already the latest version

https://crates.io/crates/proc-macro-error

Comment (sundy-li) [reply .github/actions/check/action.yml]: I will try [proc-macro-error2](https://crates.io/crates/proc-macro-error2)

**Review by everpcpc — APPROVED** [review]

**Pull request merged**